.github/workflows/inventory.yml

```diff
@@ -0,0 +1,69 @@
+name: Update Inventory
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '00 4 * * 1-5'
+
+jobs:
+  update-nodejs-inventory:
+    name: Update Node.js Engine Inventory
+    runs-on: pub-hk-ubuntu-22.04-small
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - id: install-rust-toolchain
+        name: Install Rust Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - id: install-rust-inventory-binaries
+        name: Install Rust Inventory Binaries
+        run: |
+          cargo install heroku-nodejs-utils --bin diff_versions --git https://github.com/heroku/buildpacks-nodejs
+          cargo install heroku-nodejs-utils --bin list_versions --git https://github.com/heroku/buildpacks-nodejs
+      - id: set-diff-msg
+        name: Set Diff Message
+        run: echo "::set-output name=msg::$(diff_versions node inventory/node.toml)"
+      - name: Rebuild Inventory
+        run: "list_versions node > inventory/node.toml"
+      - name: Update Changelog
+        run: echo "${{ steps.set-diff-msg.outputs.msg }}" | xargs -r -I '{}' perl -i -p -e 's/\[Unreleased\]\s+/[Unreleased]\n\n- {}/' CHANGELOG.md
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
```

Comment:
Since this is an unofficial GitHub Actions action, should we pin to an exact `X.Y.Z` version?